src/py21cmfast/src/photoncons.c

```diff
@@ -159,15 +159,15 @@ int InitialisePhotonCons(UserParams *user_params, CosmoParams *cosmo_params,
             z0 = 1./(a+delta_a) - 1.;
             z1 = 1./(a-delta_a) - 1.;
 
+            sc_i = scaling_consts_z_copy(zi,astro_params,flag_options,&sc_i,false);
```

Comment:
What does this function do? Copy over contents of a ScalingConstants struct and then update for a new redshift? If so, would a better name be something like `evolve_scaling_constants_to_redshift()`?